rust-connectors/sources/postgres/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "fluvio-connect-postgres"
+version = "0.1.0"
+edition = "2021"
+
+[[bin]]
+name = "fluvio-connect-postgres"
+path = "src/bin/main.rs"
+required-features = ["connect"]
+
+[features]
+default = ["connect"]
+connect = ["tokio", "tokio-stream", "tokio-postgres", "postgres-protocol", "tracing", "tracing-subscriber", "fluvio"]
+
+[dependencies]
+tracing = { version = "0.1", optional = true }
+tracing-subscriber = { version = "0.3", features = ["env-filter"], optional = true }
+tokio = { version = "1", features = ["full"], optional = true }
+tokio-stream = { version = "0.1", optional = true }
+tokio-postgres = { git = "https://github.com/infinyon/rust-postgres", rev = "4e86fffc7e8d4ee10c7751a58398a172a4963bfc", optional = true }
+postgres-protocol = { git = "https://github.com/infinyon/rust-postgres", rev = "4e86fffc7e8d4ee10c7751a58398a172a4963bfc", optional = true }
```

Comment:
I'm sure there's a reason we're using a github fork of the rust-postgres and tokio-postgres crates. Could you put some comments describing why. If there are github issues associated with them, link to them as well.